README.md

```diff
@@ -100,28 +100,30 @@ Clone this repository (or open in codespace), create a branch with your name, ad
+| Kevin Calva           | My favorite color is orange.                                                                                                                    |
+| Jan Niewielski        | I am the first in my family to graduate from university                                                                                         |
+| Adolfo Nava           | Male horses have 40-42 teeth while female horses could have 36-40                                                                               |
+| Jeremy Baxter         | I like saying/typing emojis instead of using them smirk emoji                                                                                   |
+| Ray Gunawidjaja       | Rice is no longer my staple food                                                                                                                |
+| Ricardo Juarez        | I used to make pastries for celebrities                                                                                                         |
+| Maya Sheriff          | I love Lucky Daye; great music!                                                                                                                 |
+| Young Song            | My biking speed is slower than running                                                                                                          |
+| Jennifer Rahman       | I love frogs!                                                                                                                                   |
+| Aleksei Borovikov     | I would love to visit Japan.                                                                                                                    |
+| Herman Contreras      | Combine blue and red to make purple                                                                                                             |
+| Brandon Varner        | Alabama's Cahaba River is known as "America's Amazon"                                                                                           |
+| Chinemelum Iloegbunam | I've been skydiving multiple times and in the proccess to get my license                                                                        |
+| Kurt Mauritz          | The Earth is always flat on a map.                                                                                                              |
+| Dylan Sarikas         | The Fun Fact about "eating 8 spiders in your sleep during your lifetime" is actually a myth                                                     |
+| Alonzo Rice           | I went to a real estate class over the week-end                                                                                                 |
+| Brian Kim             | I met Eddie Huang from Huang's world at UIC                                                                                                     |
```

Comment:
Hi Brian, this pull request diff should only have 1 line (your fun fact). Everything else should remain unchanged.

Reply:
gotcha is the spacing of the table an issue. i'm trying to make it so all the tables are the same. in noticed class 5.1 has spacing which can be attributed to linter. i see i think i got it